cfme/storage/volume.py

```diff
@@ -147,9 +147,10 @@ class VolumeAddEntities(View):
 
 
 class VolumeAddForm(View):
+    storage_manager = Select(name='storage_manager_id')
     volume_name = TextInput(name='name')
     size = TextInput(name='size')
-    tenant = BootstrapSelect(id='cloud_tenant_id')
+    tenant = Select(name='cloud_tenant_id')
```

Comment:
I'm seeing 'bootstrap-select' classes in these div's, I think these need to stay as BootstrapSelect, not Select.

Reply:
Done.